test/conformance/device/urDeviceGetInfo.cpp

```diff
@@ -105,7 +106,7 @@ static std::unordered_map<ur_device_info_t, size_t> device_info_size_map = {
     {UR_DEVICE_INFO_MAX_COMPUTE_QUEUE_INDICES, sizeof(uint32_t)},
     {UR_DEVICE_INFO_KERNEL_SET_SPECIALIZATION_CONSTANTS, sizeof(ur_bool_t)},
     {UR_DEVICE_INFO_MEMORY_BUS_WIDTH, sizeof(ur_bool_t)},
-    {UR_DEVICE_INFO_MAX_WORK_GROUPS_3D, sizeof(uint32_t)},
+    {UR_DEVICE_INFO_MAX_WORK_GROUPS_3D, sizeof(size_t) * 3u},
```

Comment:
`sizeof(size_t[3])` should also work here, then the the type in the test and the spec match up.

Reply:
Fixed in the most recent commit